lib/verifier/ed25519/keypair.c

```diff
@@ -1,5 +1,5 @@
 #include "ed25519.h"
-#include "sha3/sha3.h"
+#include "./sha3.h"
```

Comment:
Do we need `./` here? If so we should fix our include flags.

Reply:
That was due to Google coding style. They require C++ headers to always start with a directory name.

Reply:
Didn't know it required that, interesting. We should probably restructure our includes then since Google C++ style specifically outlaws use of `.` or `..` as the directory

Reply:
I will fix this in the next phase of the refactoring with the change to the install path of the SDK.